### ISSUE TITLE
The Archives are Incomplete !!!

### ISSUE THREAD

**Issue by TahsinTariq** [CodingTrain/website#2523]
In TheCodingTrain website, the [learning/nature-of-code](https://thecodingtrain.com/learning/nature-of-code/) only contains the updated versions of The Nature of Code series. But since @shiffman made older videos, those should be included in the [archives](https://thecodingtrain.com/more/archive/).

Then there are the [Intelligence and Learning Playlists](https://www.youtube.com/watch?v=y59-frfKR58&list=PLRqwX-V7Uu6bmMRCIoTi72aNWHo7epX4L) named from session - 1 to 7 ( These  are all I could find. I don't know if there's more); none of which I could find in the website. 

Navigating playlists through YouTube is confusing when there's so many videos. A page on the website would be really helpful.

**Comment by TahsinTariq** [comment]
@gruselhaus I want to add [the old nature of code playlists](https://www.youtube.com/c/TheCodingTrain/playlists?view=50&sort=dd&shelf_id=9) in the achieves. But I don't know which folder to add them in. Could you please help?

**Comment by violetcraze** [comment]
@TahsinTariq Thanks for wanting to help out this sounds awesome.

Wrote up a guide on adding new pages: https://thecodingtrain.com/Guides/pages-contribution-guide.html. This is something I've been meaning to do for awhile. I hope it's helpful and if something's unclear let me know.

I'd suggest putting them in `_more/archive/nature-of-code`. And code can go in `more/archive/nature-of-code/<page name without .md>/Processing`. That would be the folder that you save the sketch in from the processing ide. Note if you're using a different ide, processing files need to be in a directory with the same name to work right.

I think the code is at https://github.com/nature-of-code/noc-examples-processing.

Also in the case of the introduction videos change the numbering from `I.something` to `0.something`. So like `I.2` would be `0.2`.

**Comment by TahsinTariq** [comment]
@violetcraze Some playlists also contain Coding Challenges. (for example- [5: Physics  Libraries](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6akvoNKE4GAxf6ZeBYoJ4uh))

 Should I keep them? or should I exclude them?

Also, some of the links to the codes seem to be broken. Maybe they were removed/deleted. And rest of the links redirect to  `github.com/CodingTrain/website`. So, I can't find any way to contribute there.

**Comment by violetcraze** [comment]
@TahsinTariq For coding challenges we don't have a way to include a page in two places on the site. For right now exclude them, especially if they're already on the site.

I can try to find the code sometime this week. Any videos in particular you're having trouble with?

**Comment by TahsinTariq** [comment]
@violetcraze I've found some Intelligence and Learning videos in the [Neural Network playlist](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6aCibgK1PTWWu9by6XFdCfh). Do I exclude them as well?

**Comment by violetcraze** [comment]
Oh sorry I totally missed the notification for your comment. It's fine to add pages from other playlists if they don't already exist on the site. Thanks again for adding those pages!

**Comment by TahsinTariq** [comment]
@violetcraze Here's all the changes I made on my own:

* For `9.x` I changed it to `9.11`... Hope it's alright.
* Renamed  `5.0a` to `5.0.0` and `video_number: 0.0`. I've tried a lot of name variations but all of them gave me errors in jest. This is the only one I could get to work.
* I renamed `5.0b` as `5.0.1` but it still gave me an error. I don't know why. **This is the only video I couldn't add**
* I kept the Intelligence and Learning videos in there but I would like to a put them in their own page since they aren't on the website yet. Should I open a new issue and close this one? And where do I add the videos?

Really happy to help.
